tools/__init__.py

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env python3
```

Comment:
Not a python expert.
Are you going to put all python scripts under `tools` in the same package? Is it better to move block cache trace analyzer and other related scripts in the same sub directory under `tools`?

Reply:
I created block_cache_trace directory. I moved all scripts and block cache analyzer.cc into the sub directory.